railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_7_2.rb.tt

```diff
@@ -28,8 +28,8 @@
 # If `enqueue_after_transaction_commit` is set to `:default`, the queue adapter
 # will define the behaviour.
 #
-# Note: Active Job backends can disable this feature. This is generally used by
```

Comment:
I found this slightly confusing to read. `used` implies that we're referring to (enabling) the feature, but we're actually referring to the action of disabling it.